array_find.json

```diff
@@ -19,7 +19,7 @@
         },
         {
             "name": "value",
-            "description": "Value to find in `data`. If the value is `null`, this process returns always `null`.",
+            "description": "Value to find in `data`. If the value is a no-data value (or `null`), this process returns the no-data value (or `null`).",
```

Comment:
> If ... this process returns the no-data value (or `null`).

It looks a bit confusing to specify "return ... or ..." here. Why not just say "return the no-data value"? This no-data value might be null, but that depends on the context.

Reply:
See the implementation guide for an explanation.

Reply:
> **Array processes:** Some array processes (e.g. `array_find` or `any`) use `null` as a return value.
> In the context of data cube operations (e.g. in `reduce_dimension`), `null` values returned
> by the array processes shall be replaced with the no-data value of the data cube.
> As the processes may be used outside of data cubes where the no-data values are undefined,
> most processes fall back to `null` in this case (reflected through the mention of "(or `null`)" in the process description).
> This effectively means that `null` is the default value for an undefined no-data value in openEO.